src/pages/Search/SearchPage.tsx

```diff
@@ -27,6 +27,18 @@ function SearchPage({route}: SearchPageProps) {
 
     const headerContent: {[key in SearchQuery]: {icon: IconAsset; title: string}} = {
         all: {icon: Illustrations.MoneyReceipts, title: translate('common.expenses')},
+        shared: {
+            icon: Illustrations.SendMoney,
+            title: translate('common.shared'),
+        },
+        drafts: {
+            icon: Illustrations.Pencil,
+            title: translate('common.drafts'),
+        },
+        finished: {
+            icon: Illustrations.CheckmarkCircle,
+            title: translate('common.finished'),
+        },
```

Comment:
Let's follow the same code style here, i.e. either keep them to a single line or split into multiple lines

Reply:
It's automatically formatted by prettier

Reply:
Oh weird that it formats `all` differently since they all seem to have the same line length 🤔

Reply:
Okey, it occurs that it was my bad and prettier allows two options 🤷